.streamlit/config.toml

```
@@ -0,0 +1,10 @@
+# You can create a theme for your Streamlit app by creating a file called config.toml in the .streamlit directory of your project.
+[theme]
+# Choosing the colour which outlines the widgets
+primaryColor="#0000FF"
+# Background colour of the app
+backgroundColor="#FFFFFF"
+# A secondary background colour (e.g. changes the colour of the sidebar)
+secondaryBackgroundColor="#F0F2F6"
+# Colour of the text
+textColor="#262730"
```

```suggestion
textColor="#262730"
[client]
showSidebarNavigation = false
```

Here's a way to hide the navigation bar that Streamlit automatically creates!